src/components/TeamMemberStyle.js

```diff
@@ -0,0 +1,102 @@
+import { makeStyles } from '@material-ui/core/styles';
+const styles =makeStyles(theme => ({
+        heading: {
+            fontFamily: "'Catamaran', sans-serif",
+            textTransform: "uppercase",
+            color: '#000000',
+            textAlign: "center",
+        },
+        root: {
+            display: 'flex',
+            flexWrap: 'wrap',
+        },
+        "grid-container": {
+            justifyContent: 'spacing-around',
+            spacing: '5',
+            margin: '2vh'
+        },
+        dialog: {
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+        },
+        paper: {
+            padding: theme.spacing(1, 3, 2),
+            color: '#000000',
+            boxShadow: 'none'
+        },
+        "dialog-heading": {
+            fontFamily: "'Catamaran', sans-serif",
+            textTransform: "uppercase",
+            textAlign: "center",
+        },
+        "dialog-content" :{
+            fontFamily:"'Times New Roman'",
+            color: '#000000',
+            fontSize: '2rem'
```

Comment:
Good call. While we are at it, open up an enhancement to make the parent font-size responsive.